run.py

```diff
@@ -2,11 +2,10 @@
 """
 Created on Mon Feb 22 16:13:48 2021
 
-@author: adria
+@author: lore
```

Comment:
No, glia autori siamo tutti, mica solo te!
```suggestion
@author: everyone
```